src/citrine/ara/columns.py

```diff
@@ -353,3 +365,27 @@ def _attrs(self) -> List[str]:
     def __init__(self, *,
                  data_source: str):
         self.data_source = data_source
+
+
+class MolecularStructureColumn(Serializable['MolecularStructureColumn'], Column):
+    """[ALPHA] Column containing a rendering of a molecular structure value.
```

Comment:
```suggestion
    """[ALPHA] Column containing a representation of a molecular structure.
```

Not super happy about the suggestion, but rendering sounds too much like a graphics thing.  Encoding?